### PR TITLE
Homepage with Category, Videolisting Page

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Pls review asap!!!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,43 +1,73 @@
...
+  History,
+  LikedVideos,
+  Playlists,
+  WatchLater,
+  Explore,
+  LoginPage,
+  SignupPage,
+  VideoDetail,
+  Playlist,
+} from "./pages/index";
```
Comment (Bhakti199) [inline src/App.js:15]: no need to write `index` here , it will automatically refer index.

File: src/index.js [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
+                <PlaylistProvider>
+                <SinglePlaylistProvider>
+
+                  <App />
+                </SinglePlaylistProvider>
+                </PlaylistProvider>
+              </HistoryProvider>
+            </WatchLaterProvider>
+          </VideoProvider>
+        </LikeProvider>
```
Comment (Bhakti199) [inline src/index.js:39]: you may use single provider instead of using multiple provider. like main context and auth context.

**Review by Bhakti199 — APPROVED** [review]
- Code is written and structured properly.
- proper hierarchy is seen.
- No issues found.
- Provide deploy preview.

**Review by Tanjum786 — APPROVED** [review]
- folder structure looks good
- good variable naming 
- provide brief description of PR
other then all looks good to me

File: src/App.js (hunk 1 above)
Comment (MonikaJhingan) [reply src/App.js:15]: ok will do

**Pull request merged**